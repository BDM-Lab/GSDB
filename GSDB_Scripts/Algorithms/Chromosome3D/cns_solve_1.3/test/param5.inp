remark   param5.inp
remark   ========
remark  
remark   test Urey-Bradley angle terms and multiple dihedrals and 
remark   impropers 


topology
   residue test
   atom a  type=a  charge=0.0   mass=1.0 end
   atom b  type=b  charge=0.0   mass=1.0 end
   atom c  type=c  charge=0.0   mass=1.0 end
   atom d  type=d  charge=0.0   mass=1.0 end
   atom e  type=e  charge=0.0   mass=1.0 end
   atom f  type=f  charge=0.0   mass=1.0 end
   atom g  type=g  charge=0.0   mass=1.0 end
   atom h  type=h  charge=0.0   mass=1.0 end
   atom i  type=i  charge=0.0   mass=1.0 end
   atom j  type=j  charge=0.0   mass=1.0 end
   atom k  type=k  charge=0.0   mass=1.0 end
   atom l  type=l  charge=0.0   mass=1.0 end
   atom m  type=m  charge=0.0   mass=1.0 end
   atom n  type=n  charge=0.0   mass=1.0 end

   bond a    b
   bond b    c
   bond a    c

   angle c    b    a  !permutation

   dihedral  k l m n
   dihedral  a    b    c    e   multiple=4  
   dihedral  j k l m  

   improper  k l m n
   improper  b   c    a    n    multiple=3    
   improper  j k l m  

   end
end !topology

segment
  name=test
  molecule number=1 name=test end
end

parameter
   BOND A    B      100.0       1.0
   BOND B    C      100.        1.0
   BOND A    C      50.         2.0

   ANGLE A  B C       50.0     100.0  UB  50. 2.0

   DIHEdral  A B C E  MULTiple=4 80. 3 0.
                                 40. 1 180.
                                 20. 2 90.
                                 10. 4 100.
   dihedral  k l m n  0 2 180. 
   dihedral  j k l m  0 1 90.

   IMPRoper  n a c b  MULTiple=3 70. 2 0.
                                 10. 0 180.
                                  5. 0 90.
   improper  k l m n  0 2 180. 
   improper  j k l m  0 1 90.
   ?

   verbose

end !parameter

coor
ATOM      1  A   TEST    1       0.000   0.000   2.000  1.00  0.00      TEST
ATOM      2  B   TEST    1       1.000   0.000   1.000  1.00  0.00      TEST
ATOM      3  C   TEST    1      -1.000   1.000   0.000  1.00  0.00      TEST
ATOM      4  D   TEST    1       2.000   1.000   0.000  1.00  0.00      TEST
ATOM      5  E   TEST    1       0.000   1.000   0.000  1.00  0.00      TEST
ATOM      6  F   TEST    1       1.000   1.000   0.000  1.00  0.00      TEST
ATOM      7  G   TEST    1       0.000  -1.000   0.000  1.00  0.00      TEST
ATOM      8  H   TEST    1       1.000  -1.000   0.000  1.00  0.00      TEST
ATOM      9  I   TEST    1      -1.000  -1.000   0.000  1.00  0.00      TEST
ATOM     10  J   TEST    1       2.000   2.000   0.000  1.00  0.00      TEST
ATOM     11  K   TEST    1       3.000   3.000   0.000  1.00  0.00      TEST
ATOM     12  L   TEST    1       4.000   2.000   0.000  1.00  0.00      TEST
ATOM     13  M   TEST    1       5.000   1.000   0.000  1.00  0.00      TEST
ATOM     14  N   TEST    1       6.000   1.000   0.000  1.00  0.00      TEST
END

flags exclude * include bonds angle dihedrals impropers end

energy end

pick bond ( name a ) ( name c ) energy 
pick angle ( name a ) ( name b ) ( name c ) geometry
pick angle ( name a ) ( name b ) ( name c ) energy
print angles

parameter 
   angle ( name a ) ( name b ) ( name c )  TOKEN TOKEN  UB 0. 0.
   bond ( name a ) ( name b ) 0. TOKEN
   bond ( name b ) ( name c ) 0. TOKEN
end

{* test total-angle = angle + ub-bond *}
energy end

parameter
   angle ( name a ) ( name b ) ( name c )  TOKEN TOKEN  UB 10. 1.
   bond ( name a ) ( name c ) 10. 1.
end
energy end

pick bond ( name a ) ( name c ) energy 
pick angle ( name a ) ( name b ) ( name c ) geometry
pick angle ( name a ) ( name b ) ( name c ) energy
print angles


pick dihedral ( name A ) ( name B ) ( name C ) ( name E ) geometry
pick dihedral ( name A ) ( name B ) ( name C ) ( name E ) energy
pick dihedral ( name j ) ( name k ) ( name l ) ( name m ) energy
print dihedrals

pick improper ( name N ) ( name A ) ( name C ) ( name B ) geometry
pick improper ( name N ) ( name A ) ( name C ) ( name B ) energy
print impropers

parameter

   dihedral ( name A ) ( name B ) ( name C ) ( name E ) multiple=4
                 10. 0  180.
                  5. 1   90.
                  2. 2   70.
                  1. 3   30.

   improper ( name N ) ( name A ) ( name C ) ( name B ) multiple=3
                TOKEN TOKEN 1.
                TOKEN TOKEN 2.
                TOKEN TOKEN 3.
end
print dihedrals
print impropers

parameter
  reduce mode=average end
  ?
end

parameter
  learn init selection=( all ) mode=nostatistics end
  learn accu end
  learn terminate end
  reduce mode=average end
  ?
end

  
stop
