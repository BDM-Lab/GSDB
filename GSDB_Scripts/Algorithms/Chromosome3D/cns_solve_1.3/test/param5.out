          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:05 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>remark   param5.inp 
 CNSsolve>remark   ======== 
 CNSsolve>remark 
 CNSsolve>remark   test Urey-Bradley angle terms and multiple dihedrals and 
 CNSsolve>remark   impropers 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>topology 
 RTFRDR>   residue test 
 RESIDUE>   atom a  type=a  charge=0.0   mass=1.0 end 
 RESIDUE>   atom b  type=b  charge=0.0   mass=1.0 end 
 RESIDUE>   atom c  type=c  charge=0.0   mass=1.0 end 
 RESIDUE>   atom d  type=d  charge=0.0   mass=1.0 end 
 RESIDUE>   atom e  type=e  charge=0.0   mass=1.0 end 
 RESIDUE>   atom f  type=f  charge=0.0   mass=1.0 end 
 RESIDUE>   atom g  type=g  charge=0.0   mass=1.0 end 
 RESIDUE>   atom h  type=h  charge=0.0   mass=1.0 end 
 RESIDUE>   atom i  type=i  charge=0.0   mass=1.0 end 
 RESIDUE>   atom j  type=j  charge=0.0   mass=1.0 end 
 RESIDUE>   atom k  type=k  charge=0.0   mass=1.0 end 
 RESIDUE>   atom l  type=l  charge=0.0   mass=1.0 end 
 RESIDUE>   atom m  type=m  charge=0.0   mass=1.0 end 
 RESIDUE>   atom n  type=n  charge=0.0   mass=1.0 end 
 RESIDUE> 
 RESIDUE>   bond a    b 
 RESIDUE>   bond b    c 
 RESIDUE>   bond a    c 
 RESIDUE> 
 RESIDUE>   angle c    b    a  !permutation 
 RESIDUE> 
 RESIDUE>   dihedral  k l m n 
 RESIDUE>   dihedral  a    b    c    e   multiple=4 
 RESIDUE>   dihedral  j k l m 
 RESIDUE> 
 RESIDUE>   improper  k l m n 
 RESIDUE>   improper  b   c    a    n    multiple=3 
 RESIDUE>   improper  j k l m 
 RESIDUE> 
 RESIDUE>   end 
 RTFRDR>end !topology 
 CNSsolve> 
 CNSsolve>segment 
 SEGMENT>  name=test 
 SEGMENT>  molecule number=1 name=test end 
 SEGMENT>end 
 Status of internal molecular topology database:
 -> NATOM=         14(MAXA=     1000000)  NBOND=          3(MAXB=     1000000)
 -> NTHETA=         1(MAXT=     2000000)  NGRP=           1(MAXGRP=   1000000)
 -> NPHI=           6(MAXP=     2000000)  NIMPHI=         5(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve> 
 CNSsolve>parameter 
 PARRDR>   BOND A    B      100.0       1.0 
 PARRDR>   BOND B    C      100.        1.0 
 PARRDR>   BOND A    C      50.         2.0 
 PARRDR> 
 PARRDR>   ANGLE A  B C       50.0     100.0  UB  50. 2.0 
 PARRDR> 
 PARRDR>   DIHEdral  A B C E  MULTiple=4 80. 3 0. 
 energy-constant=                                 40. 1 180. 
 energy-constant=                                 20. 2 90. 
 energy-constant=                                 10. 4 100. 
 PARRDR>   dihedral  k l m n  0 2 180. 
 PARRDR>   dihedral  j k l m  0 1 90. 
 PARRDR> 
 PARRDR>   IMPRoper  n a c b  MULTiple=3 70. 2 0. 
 energy-constant=                                 10. 0 180. 
 energy-constant=                                  5. 0 90. 
 PARRDR>   improper  k l m n  0 2 180. 
 PARRDR>   improper  j k l m  0 1 90. 
 PARRDR>   ? 

 BOND  A    B        100.000 {sd=     0.054}      1.000
 BOND  A    C         50.000 {sd=     0.077}      2.000
 BOND  B    C        100.000 {sd=     0.054}      1.000

 ANGLe  A    B    C         50.00 {sd=     4.409}    100.0000    UB     50.000       2.000

 DIHEdral  A    B    C    E      MULTiple= 4      80.00     3      0.0000
                                                  40.00     1    180.0000
                                                  20.00     2     90.0000
                                                  10.00     4    100.0000
 DIHEdral  J    K    L    M           0.00 {sd=999999.000}    1     90.0000
 DIHEdral  K    L    M    N           0.00 {sd=999999.000}    2    180.0000

 IMPRoper  B    C    A    N      MULTiple= 3      70.00     2      0.0000
                                                  10.00     0    180.0000
                                                   5.00     0     90.0000
 IMPRoper  J    K    L    M           0.00 {sd=999999.000}    1     90.0000
 IMPRoper  K    L    M    N           0.00 {sd=999999.000}    2    180.0000
 PARRDR> 
 PARRDR>   verbose 
 Listing of all atom-based (F) and type-based (T) parameters.

 BOND (ATOM TEST 1    A   ) (ATOM TEST 1    B   )    100.000T      1.000T
 BOND (ATOM TEST 1    B   ) (ATOM TEST 1    C   )    100.000T      1.000T
 BOND (ATOM TEST 1    A   ) (ATOM TEST 1    C   )     50.000T      2.000T

 ANGL (ATOM TEST 1    C   ) (ATOM TEST 1    B   ) 
                            (ATOM TEST 1    A   )     50.000T    100.000T  UB     50.000T       2.000T

 DIHE (ATOM TEST 1    K   ) (ATOM TEST 1    L   ) 
      (ATOM TEST 1    M   ) (ATOM TEST 1    N   )       MULT   1
                                                        0.00T   2T     180.00T
 DIHE (ATOM TEST 1    A   ) (ATOM TEST 1    B   ) 
      (ATOM TEST 1    C   ) (ATOM TEST 1    E   )       MULT   4
                                                       80.00T   3T       0.00T
                                                       40.00T   1T     180.00T
                                                       20.00T   2T      90.00T
                                                       10.00T   4T     100.00T
 DIHE (ATOM TEST 1    J   ) (ATOM TEST 1    K   ) 
      (ATOM TEST 1    L   ) (ATOM TEST 1    M   )       MULT   1
                                                        0.00T   1T      90.00T

 IMPR (ATOM TEST 1    K   ) (ATOM TEST 1    L   ) 
      (ATOM TEST 1    M   ) (ATOM TEST 1    N   )       MULT   1
                                                        0.00T   2T     180.00T
 IMPR (ATOM TEST 1    B   ) (ATOM TEST 1    C   ) 
      (ATOM TEST 1    A   ) (ATOM TEST 1    N   )       MULT   3
                                                       70.00T   2T       0.00T
                                                       10.00T   0T     180.00T
                                                        5.00T   0T      90.00T
 IMPR (ATOM TEST 1    J   ) (ATOM TEST 1    K   ) 
      (ATOM TEST 1    L   ) (ATOM TEST 1    M   )       MULT   1
                                                        0.00T   1T      90.00T
 PARRDR> 
 PARRDR>end !parameter 
 CNSsolve> 
 CNSsolve>coor 
 COOR>ATOM      1  A   TEST    1       0.000   0.000   2.000  1.00  0.00      TEST 
 COOR>ATOM      2  B   TEST    1       1.000   0.000   1.000  1.00  0.00      TEST 
 CNSsolve> 
 CNSsolve>flags exclude * include bonds angle dihedrals impropers end 
 CNSsolve> 
 CNSsolve>energy end 
 --------------- cycle=     1 --------------------------------------------------
 | Etotal =686.530    grad(E)=115.390    E(BOND)=237.361    E(ANGL)=21.024     |
 | E(DIHE)=217.122    E(IMPR)=211.023                                          |
 -------------------------------------------------------------------------------
 CNSsolve> 
 CNSsolve>pick bond ( name a ) ( name c ) energy 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 ENER=     10.102051
 CNSsolve>pick angle ( name a ) ( name b ) ( name c ) geometry 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 GEOM=     73.221345
 CNSsolve>pick angle ( name a ) ( name b ) ( name c ) energy 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 ENER=     21.024054
 CNSsolve>print angles 
 (atom-i        |atom-j        |atom-k        )  angle    equil.     delta    energy  const. 

 (TEST 1    C   |TEST 1    B   |TEST 1    A   )   73.221  100.000  -26.779   10.922   50.000
                        Urey-Bradley bond term:    2.449    2.000    0.449   10.102   50.000
 RMS deviation=   0.449
 CNSsolve> 
 CNSsolve>parameter 
 PARRDR>   angle ( name a ) ( name b ) ( name c )  TOKEN TOKEN  UB 0. 0. 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 PARRDR>   bond ( name a ) ( name b ) 0. TOKEN 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 PARRDR>   bond ( name b ) ( name c ) 0. TOKEN 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 PARRDR>end 
 CNSsolve> 
 CNSsolve>{* test total-angle = angle + ub-bond *} 
 CNSsolve>energy end 
 --------------- cycle=     2 --------------------------------------------------
 | Etotal =449.169    grad(E)=79.883     E(BOND)=10.102     E(ANGL)=10.922     |
 | E(DIHE)=217.122    E(IMPR)=211.023                                          |
 -------------------------------------------------------------------------------
 CNSsolve> 
 CNSsolve>parameter 
 PARRDR>   angle ( name a ) ( name b ) ( name c )  TOKEN TOKEN  UB 10. 1. 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 PARRDR>   bond ( name a ) ( name c ) 10. 1. 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 PARRDR>end 
 CNSsolve>energy end 
 --------------- cycle=     3 --------------------------------------------------
 | Etotal =481.087    grad(E)=80.401     E(BOND)=21.010     E(ANGL)=31.932     |
 | E(DIHE)=217.122    E(IMPR)=211.023                                          |
 -------------------------------------------------------------------------------
 CNSsolve> 
 CNSsolve>pick bond ( name a ) ( name c ) energy 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 ENER=     21.010205
 CNSsolve>pick angle ( name a ) ( name b ) ( name c ) geometry 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 GEOM=     73.221345
 CNSsolve>pick angle ( name a ) ( name b ) ( name c ) energy 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 ENER=     31.932207
 CNSsolve>print angles 
 (atom-i        |atom-j        |atom-k        )  angle    equil.     delta    energy  const. 

 (TEST 1    C   |TEST 1    B   |TEST 1    A   )   73.221  100.000  -26.779   10.922   50.000
                        Urey-Bradley bond term:    2.449    1.000    1.449   21.010   10.000
 RMS deviation=   1.449
 CNSsolve> 
 CNSsolve> 
 CNSsolve>pick dihedral ( name A ) ( name B ) ( name C ) ( name E ) geometry 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 GEOM=    148.517846
 CNSsolve>pick dihedral ( name A ) ( name B ) ( name C ) ( name E ) energy 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 ENER=    217.122183
 CNSsolve>pick dihedral ( name j ) ( name k ) ( name l ) ( name m ) energy 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 ENER=      0.000000
 CNSsolve>print dihedrals 
 (atom-i        |atom-j        |atom-k        |atom-L        )    angle    equil.   delta    energy   const.   period

 (TEST 1    K   |TEST 1    L   |TEST 1    M   |TEST 1    N   )  -90.000    0.000   90.000    0.000    0.000   2
 (TEST 1    A   |TEST 1    B   |TEST 1    C   |TEST 1    E   )  148.518  180.000   31.482   86.202   80.000   3
 (TEST 1    A   |TEST 1    B   |TEST 1    C   |TEST 1    E   )  148.518    0.000 -148.518   74.112   40.000   1
 (TEST 1    A   |TEST 1    B   |TEST 1    C   |TEST 1    E   )  148.518  225.000   76.482   37.814   20.000   2
 (TEST 1    A   |TEST 1    B   |TEST 1    C   |TEST 1    E   )  148.518  110.000  -38.518   18.993   10.000   4
 (TEST 1    J   |TEST 1    K   |TEST 1    L   |TEST 1    M   )  -90.000   90.000  180.000    0.000    0.000   1
 RMS deviation= 108.691
 CNSsolve> 
 CNSsolve>pick improper ( name N ) ( name A ) ( name C ) ( name B ) geometry 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 GEOM=     19.286325
 CNSsolve>pick improper ( name N ) ( name A ) ( name C ) ( name B ) energy 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 ENER=    211.022637
 CNSsolve>print impropers 
 (atom-i        |atom-j        |atom-k        |atom-L        )    angle    equil.   delta    energy   const.   period

 (TEST 1    K   |TEST 1    L   |TEST 1    M   |TEST 1    N   )  -90.000    0.000   90.000    0.000    0.000   2
 (TEST 1    B   |TEST 1    C   |TEST 1    A   |TEST 1    N   )   19.286   90.000   70.714  124.727   70.000   2
 (TEST 1    B   |TEST 1    C   |TEST 1    A   |TEST 1    N   )   19.286  180.000  160.714   78.679   10.000   0
 (TEST 1    B   |TEST 1    C   |TEST 1    A   |TEST 1    N   )   19.286   90.000   70.714    7.616    5.000   0
 (TEST 1    J   |TEST 1    K   |TEST 1    L   |TEST 1    M   )  -90.000   90.000  180.000    0.000    0.000   1
 RMS deviation= 123.555
 CNSsolve> 
 CNSsolve>parameter 
 PARRDR> 
 PARRDR>   dihedral ( name A ) ( name B ) ( name C ) ( name E ) multiple=4 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 energy-constant=                 10. 0  180. 
 energy-constant=                  5. 1   90. 
 energy-constant=                  2. 2   70. 
 energy-constant=                  1. 3   30. 
 PARRDR> 
 PARRDR>   improper ( name N ) ( name A ) ( name C ) ( name B ) multiple=3 
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 SELRPN:      1 atoms have been selected out of     14
 energy-constant=                TOKEN TOKEN 1. 
 energy-constant=                TOKEN TOKEN 2. 
 energy-constant=                TOKEN TOKEN 3. 
 PARRDR>end 
 CNSsolve>print dihedrals 
 (atom-i        |atom-j        |atom-k        |atom-L        )    angle    equil.   delta    energy   const.   period

 (TEST 1    K   |TEST 1    L   |TEST 1    M   |TEST 1    N   )  -90.000    0.000   90.000    0.000    0.000   2
 (TEST 1    A   |TEST 1    B   |TEST 1    C   |TEST 1    E   )  148.518  180.000   31.482    3.019   10.000   0
 (TEST 1    A   |TEST 1    B   |TEST 1    C   |TEST 1    E   )  148.518   90.000  -58.518    2.389    5.000   1
 (TEST 1    A   |TEST 1    B   |TEST 1    C   |TEST 1    E   )  148.518  235.000   86.482    3.985    2.000   2
 (TEST 1    A   |TEST 1    B   |TEST 1    C   |TEST 1    E   )  148.518  170.000   21.482    0.569    1.000   3
 (TEST 1    J   |TEST 1    K   |TEST 1    L   |TEST 1    M   )  -90.000   90.000  180.000    0.000    0.000   1
 RMS deviation=  93.858
 CNSsolve>print impropers 
 (atom-i        |atom-j        |atom-k        |atom-L        )    angle    equil.   delta    energy   const.   period

 (TEST 1    K   |TEST 1    L   |TEST 1    M   |TEST 1    N   )  -90.000    0.000   90.000    0.000    0.000   2
 (TEST 1    B   |TEST 1    C   |TEST 1    A   |TEST 1    N   )   19.286   89.500   70.214  123.957   70.000   2
 (TEST 1    B   |TEST 1    C   |TEST 1    A   |TEST 1    N   )   19.286    2.000  -17.286    0.910   10.000   0
 (TEST 1    B   |TEST 1    C   |TEST 1    A   |TEST 1    N   )   19.286    3.000  -16.286    0.404    5.000   0
 (TEST 1    J   |TEST 1    K   |TEST 1    L   |TEST 1    M   )  -90.000   90.000  180.000    0.000    0.000   1
 RMS deviation=  95.910
 CNSsolve> 
 CNSsolve>parameter 
 PARRDR>  reduce mode=average end 
 PARRDR>  ? 

 BOND  A    B          0.000 {sd=999999.000}      1.000
 BOND  A    C         10.000 {sd=     0.172}      1.000
 BOND  B    C          0.000 {sd=999999.000}      1.000

 ANGLe  A    B    C         50.00 {sd=     4.409}    100.0000    UB     10.000       1.000

 DIHEdral  A    B    C    E      MULTiple= 4      10.00     0    180.0000
                                                   5.00     1     90.0000
                                                   2.00     2     70.0000
                                                   1.00     3     30.0000
 DIHEdral  J    K    L    M           0.00 {sd=999999.000}    1     90.0000
 DIHEdral  K    L    M    N           0.00 {sd=999999.000}    2    180.0000

 IMPRoper  B    C    A    N      MULTiple= 3      70.00     2      1.0000
                                                  10.00     0      2.0000
                                                   5.00     0      3.0000
 IMPRoper  J    K    L    M           0.00 {sd=999999.000}    1     90.0000
 IMPRoper  K    L    M    N           0.00 {sd=999999.000}    2    180.0000
 PARRDR>end 
 CNSsolve> 
 CNSsolve>parameter 
 PARRDR>  learn init selection=( all ) mode=nostatistics end 
 SELRPN:     14 atoms have been selected out of     14
 PARRDR>  learn accu end 
 PARRDR>  learn terminate end 
 PARRDR>  reduce mode=average end 
 PARRDR>  ? 

 BOND  A    B          0.000 {sd=999999.000}      1.414
 BOND  A    C         10.000 {sd=     0.172}      2.449
 BOND  B    C          0.000 {sd=999999.000}      2.449

 ANGLe  A    B    C         50.00 {sd=     4.409}     73.2213

 DIHEdral  A    B    C    E      MULTiple= 4      10.00     0    148.5178
                                                   5.00     0    148.5178
                                                   2.00     0    148.5178
                                                   1.00     0    148.5178
 DIHEdral  J    K    L    M           0.00 {sd=999999.000}    0    -90.0000
 DIHEdral  K    L    M    N           0.00 {sd=999999.000}    0    -90.0000

 IMPRoper  B    C    A    N      MULTiple= 3      70.00     0     19.2863
                                                  10.00     0     19.2863
                                                   5.00     0     19.2863
 IMPRoper  J    K    L    M           0.00 {sd=999999.000}    0    -90.0000
 IMPRoper  K    L    M    N           0.00 {sd=999999.000}    0    -90.0000
 PARRDR>end 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8250480 bytes
           Maximum dynamic memory overhead:           288 bytes
           Program started at: 16:08:05 on 05-Jul-2018
           Program stopped at: 16:08:05 on 05-Jul-2018
           CPU time used:       0.0088 seconds
          ============================================================
